src/screens/CurrencySelectModal.js

```diff
@@ -323,9 +323,12 @@ export default function CurrencySelectModal() {
               isFetching={loadingAllTokens}
               isSearching={isSearching}
               onChangeText={setSearchQuery}
-              onFocus={handleFocus}
               ref={searchInputRef}
               searchQuery={searchQuery}
+              {...(ios && {
+                onFocus: handleFocus,
+                ref: searchInputRef,
+              })}
```

Comment:
Spreading something that returns false can randomly break on android

```suggestion
              {...(ios ? {
                onFocus: handleFocus,
                ref: searchInputRef,
              } : {})}
```